resolwe_api/resolwe.py

```diff
@@ -35,21 +37,15 @@ def __init__(self, email=DEFAULT_EMAIL, password=DEFAULT_PASSWD, url=DEFAULT_URL
         """
         Connect to Resolwe on desired url and provide credentials.
         """
-
         self.url = url
         self.auth = ResAuth(email, password, url)
         self.api = slumber.API(urlparse.urljoin(url, '/api/'), self.auth, append_slash=False)
 
-
-        # Defines cache to reuse the data already downloaded.
-        #
-        # data = dict of (data_id - Data) pairs
-        # collections = dict of (collection_id - collection) pairs
-        # collections_data - dict of (collection_id - data_list) pairs,
-        #   where data_list is list of all data objects for given collection_id
+        # data = dict of (data_id - Data object) pairs
+        # collections = dict of (collection_id - Collection object) pairs
+        # collections_data - dict of (collection_id - Data objects list) pairs,
         self.cache = {'data': {}, 'collections': None, 'collections_data': {}}
-        # How about when the sever is changed - how can we get the changes? For example someone adds a collection or some data. Cache does not know this and cannot update?
-        # this cache thing is just partially implemented now...
+        # TODO: what if there are updates on server? How to know when to update cache?
```

Comment:
It would be great if we could use a 3rd-party library for caching, like [requests-cache](https://pypi.python.org/pypi/requests-cache). This would really reduce the complexity of our code and lower the chance of error.